chat-widget/src/components/chatbuttonstateful/ChatButtonStateful.tsx

```diff
@@ -35,7 +37,13 @@ export const ChatButtonStateful = (props: IChatButtonStatefulParams) => {
             TelemetryHelper.logActionEvent(LogLevel.INFO, {
                 Event: TelemetryEvent.LCWChatButtonClicked
             });
-            if (state.appStates.isMinimized) {
+            if (state.appStates.proactiveChatStates.proactiveChatInNewWindow) {
```

Comment:
Do we need null check for "proactiveChatStates" here?

Reply:
We don't, these are always defined